### PR TITLE
Update 2019.md

### PR DESCRIPTION
**Pull request by hughhhh**

### CODE REVIEW

File: 2019.md [hunk 1]
```diff
@@ -2,33 +2,40 @@
...
-- [ ] Drive a AMG GTR
-- [ ] Shoot a film with the creativesnextdoor
-- [ ] Release a podcast
-- [ ] Top 5 contributor in Superset
-- [ ] Fluent in Spanish
 
+### Main Objectives 🔑
+- [ ] 100 commits into Superset
+- [ ] 🚢 an app into the App Store
+- [ ] Shoot a short film
```
Comment (hughhhh) [inline 2019.md]: pick between 2 of these

File: 2019.md [hunk 2]
```diff
@@ -2,33 +2,45 @@
...
+- [ ] Fluent in Spanish
```
Comment (hamt3ch) [inline 2019.md]: Check this video out:
https://www.youtube.com/channel/UC9yudInUYzMh9H4gJs4DrHg

File: 2019.md (hunk 1 above)
Comment (psdewar2) [reply 2019.md]: where is release a podcast haha
do a cost benefit analysis on each objective and start with the ones you can start easily and get the most out of

File: 2019.md [hunk 3]
```diff
@@ -2,33 +2,45 @@
...
+- [ ] Play in a basketball league
 
-### Skills
-- [ ]
+#### Skills
+- [ ] Learn how to use protobufs
 
-### Mental
+#### Mental
 - [ ] Read 10 books
```
Comment (psdewar2) [inline 2019.md]: my boy is tryna read 27 books before 27. He said he just reads and retains whatever he can from that one read

File: 2019.md [hunk 4]
```diff
@@ -2,33 +2,45 @@
...
+#### Career
 - [ ] Release a blog under Lyft's Engineering Organization
+  - [ ] Our Interface Design Language at Lyft
+  - [ ] Dashboarding at Lyft (StacheDash x DDR project launch)
+  - [ ] Using Protbufs for codgen service creation
+- [ ] Give a conference talk
+  - [ ] Dashboarding at Lyft (StacheDash x DDR project launch)
+  - [ ] Using Protbufs for codgen service creation
+- [ ] Publish an article on pudding.cool
+
```
Comment (psdewar2) [inline 2019.md]:
```suggestion
 #### Faith
 - Prayer
 - Meditation
```

File: 2019.md (hunk 3 above)
Comment (hughhhh) [reply 2019.md]: so you are saying to dumb down the list?

Comment (psdewar2) [reply 2019.md]: not at all, but you should determine whether you would like to master a couple things or just read for leisure and general knowledge. both are equally valuable

File: 2019.md [hunk 5]
```diff
@@ -2,33 +2,45 @@
...
+#### Mental
 - [ ] Read 10 books
-
-### Health
-- [ ] 50 Pushups in 60secs
-- [ ] 20 Pullups
-- [ ] Power Clean 200lb
+  - [ ] Pragmatic Programmer
+  - [ ] The Subtle Art of Not Giving a F*ck
+  - [ ] Design Patterns (Tech Book)
```
Comment (hamt3ch) [inline 2019.md]: Add "Think and Grow Rich" - Napoleon Hill

File: 2019.md [hunk 6]
```diff
@@ -37,19 +38,18 @@ be focusing of 4 specific goals and outline how I plan to accomplish them over t
...
-
 #### Mental
 - [ ] Read 10 books
   - [ ] Pragmatic Programmer
   - [ ] The Subtle Art of Not Giving a F*ck
   - [ ] Design Patterns (Tech Book)
   - [ ] Think and Grow Rich
 - [ ] Meditation
-  - [ ] 
+  - [ ] Yoga once a week (Wednesday nights)
```
Comment (psdewar2) [inline 2019.md]: It would helpful to meditate daily. Look up the app Headspace.

File: 2019.md [hunk 7]
```diff
@@ -37,19 +38,18 @@ be focusing of 4 specific goals and outline how I plan to accomplish them over t
...
 
 #### Faith
- [ ] Prayer
- 
+- [ ] Prayer
+- [ ]
```
Comment (psdewar2) [inline 2019.md]: It would also be helpful to pray daily.

File: 2019.md [hunk 8]
```diff
@@ -12,10 +12,10 @@ be focusing of 4 specific goals and outline how I plan to accomplish them over t
   - [ ] Peep
 - [ ] Shoot a short film
   - [ ] locale 2
-  - [ ] Making Peep
+  - [ ] hnd1 - lupe
   - [ ] Music Video
   - [ ] Rapbook promo video
-- [ ] Fluent in Spanish
+- [ ] Fluent in French
```
Comment (psdewar2) [inline 2019.md]: where is spanish

File: 2019.md [hunk 9]
```diff
@@ -12,10 +12,10 @@ be focusing of 4 specific goals and outline how I plan to accomplish them over t
   - [ ] Peep
 - [ ] Shoot a short film
   - [ ] locale 2
-  - [ ] Making Peep
+  - [ ] hnd1 - lupe
```
Comment (psdewar2) [inline 2019.md]: what is lupe

File: 2019.md (hunk 8 above)
Comment (hughhhh) [reply 2019.md]: pivoting to french since i need it when i open up my clothing brand

File: 2019.md (hunk 9 above)
Comment (hughhhh) [reply 2019.md]: peep rebrand -> lupe

**Pull request merged**